include/nbl/asset/IMeshPacker.h

```diff
@@ -78,8 +32,23 @@ class MeshPackerBase
     };
```

Comment:
what do you need this struct for?

Reply:
v1 stuff
https://github.com/Przemog1/Nabla/blob/mesh_packer/include/nbl/asset/IMeshPacker.h#L141